jsconfig-lint.json

```diff
@@ -0,0 +1,4 @@
+{
+  "extends": "./jsconfig.json",
+  "include": ["src/**/*"]
+}
```

Comment:
Just curious: what's the need for this? Wouldn't the base config also include `src/**/*`, given the default value for `include` is "everything" and `src` hasn't been explicitly excluded?

Reply:
Yup, that's right. It does. I created this file to *only* include `./src/**/*` and so other src directories (like compat and hooks) and test directories are not included.

Running `tsc -p jsconfig.json` results in >5k errors. I wanted to start small and just enforce type checks on the root source folder for now.

In other words - this is the config we'll use for CI where we only enforce type correctness on the root src folder, not nested src folders, yet.

Reply:
> Running tsc -p jsconfig.json results in >5k errors.

Oh, lol. That makes sense. I forgot that I turn on my blinders when it comes to TS in this repo.